internal/utils/utils_test.go

```diff
@@ -372,8 +371,6 @@ func TestDownloadFile(t *testing.T) {
 	f, err := DownloadFile("https://www.elastic.co/robots.txt")
 	assert.Nil(t, err)
 	defer os.Remove(filepath.Dir(f))
-
-	assert.True(t, strings.HasSuffix(f, "robots.txt"))
```

Comment:
The downloaded filename is now an UUID, so that we cannot make an assertion for its name